ui/src/routes/devices.$id.tsx

```diff
@@ -34,27 +38,21 @@ import {
   VideoState,
 } from "@/hooks/stores";
 import WebRTCVideo from "@components/WebRTCVideo";
-import { checkAuth, isInCloud, isOnDevice } from "@/main";
 import DashboardNavbar from "@components/Header";
-import ConnectionStatsSidebar from "@/components/sidebar/connectionStats";
-import { JsonRpcRequest, useJsonRpc } from "@/hooks/useJsonRpc";
-import Terminal from "@components/Terminal";
-import { CLOUD_API, DEVICE_API } from "@/ui.config";
-
-import UpdateInProgressStatusCard from "../components/UpdateInProgressStatusCard";
-import api from "../api";
-import Modal from "../components/Modal";
-import { useDeviceUiNavigation } from "../hooks/useAppNavigation";
+const ConnectionStatsSidebar = lazy(() => import('@/components/sidebar/connectionStats'));
```

Comment:
These sub-function on the main device page are not always active, so making them lazy lets Vite pull their code out to separate JS modules. This is a huge gain for Terminal and ConnectionStatsSidebar because they're not usually active.